### DISCUSSION TITLE
create view x if not exist does not work

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#5677] [General]
### What happens?

create view x if not exist does not work

### To Reproduce

create view x if not exist as select * from foo does not work

### OS:

windows 10

### DuckDB Version:

0.6.1

### DuckDB Client:

Python

### Full Name:

mimoune

### Affiliation:

Personal project

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by hannes** [comment]
That's a feature request not a bug, right?

**Comment by Mytherin** [comment]
Implemented in #5682

**Reply by djouallah** [reply]
thanks @Mytherin
